activesupport/lib/active_support/testing/assertions.rb

```diff
@@ -30,6 +30,8 @@ def assert_not(object, message = nil)
       #   end
       def assert_nothing_raised
         yield
+      rescue => error
+        raise Minitest::UnexpectedError.new(error)
```

Comment:
This made `minitest` a surprised dependency for 6.1

Reply:
Oh good pickup. I will have a go at fixing next week unless someone beats me to it.

Reply:
Oh wait I thought this was https://github.com/rails/rails/pull/42459

That said isn’t minitest already a dependency? https://github.com/rails/rails/blob/9c77d72ecc7fdf490e778b0807882fd53115e7a3/activesupport/lib/active_support/test_case.rb#L4

Reply:
Yes it is: https://github.com/rails/rails/blob/9c77d72ecc7fdf490e778b0807882fd53115e7a3/activesupport/activesupport.gemspec#L40

So not sure what @mathieujobin means exactly.